WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/eclipse/ui/wizards/projectregistration/ProjectCreatedEndingPage.java

```diff
@@ -132,6 +133,17 @@ private void createPageContent() {
 		}
 	}
 
+	/**
+	 * Shows the label and link to ask the new user to fill out the survey on
+	 * debugging.
+	 */
+	private void createDebugSurveyInfo() {
+		UIUtils.createBoldLabel(
+				"Do you ever debug? Did you know WatchDog now also reports on debugging?",
```

Comment:
@nspruit I accepted this already, but you can extract the string to nl.tudelft.watchdog.core.util.WatchDogGlobals?